src/Illuminate/Auth/TokenGuard.php

```diff
@@ -18,11 +18,11 @@ class TokenGuard implements Guard
     protected $request;
 
     /**
-     * The name of the field on the request containing the API token.
+     * The name of the query string item from the request containing the API token.
      *
      * @var string
      */
-    protected $inputKey;
+    protected $queryKey;
```

Comment:
No, you need to stick with the old variable name or send this request to `master` branch (for Laravel 5.4), breaking BC is only permitted on major release.